src/keys.cc

```diff
@@ -165,13 +188,10 @@ String key_to_str(Key key)
     else
         res = String{key.key};
 
-    switch (key.modifiers)
-    {
-    case Key::Modifiers::Control:    res = "c-" + res; named = true; break;
-    case Key::Modifiers::Alt:        res = "a-" + res; named = true; break;
-    case Key::Modifiers::ControlAlt: res = "c-a-" + res; named = true; break;
-    default: break;
-    }
+    if (key.modifiers & Key::Modifiers::Shift)   { res = "s-" + res; named = true; }
```

Comment:
might be nitpicking, but I'd put them in the control, alt, shift order, as I tend to say "ctrl alt blah", "ctrl shift blah" "alt shift blah" rather than "alt ctrl blah" "shift ctrl blah" or "shift alt blah".

Reply:
You are right, I was confused.